TASK-53005: In Tasks application, updating name project results loss of project.

IlhemEssaadi opened the pull request:
ISSUES : In Tasks application, when updating the project name, the updates are not immediately made and the display of the project is lost after refreshing.
FIX : the problem is in the condition of reset method which is launched when we refresh the page and it fixed by fixing the condition and the second problem is that the manager and participator variables get the wrong value, so it is fixed by allow manager and participator variables to get the value of GroupSpace.

IlhemEssaadi committed:
TASK-53005: In Tasks application, updating name project results loss of project. (#745)

ISSUES : In Tasks application, when updating the project name, the updates are not immediately made and the display of the project is lost after refreshing.
FIX : the problem is in the condition of reset method which is launched when we refresh the page and it fixed by fixing the condition and the second problem is that the manager and participator variables get the wrong value, so it is fixed by allow manager and participator variables to get the value of GroupSpace.

boubaker commented on task-management/src/main/webapp/vue-app/tasks-management/components/Project/AddProjectDrawer.vue:
```diff
@@ -265,8 +265,8 @@ export default {
-                projects.manager.push(`manager:/spaces/${manager_el.remoteId}`);
-                projects.participator.push(`member:/spaces/${manager_el.remoteId}`);
+                projects.manager.push(`manager:/spaces/${eXo.env.portal.spaceGroup}`);
+                projects.participator.push(`member:/spaces/${eXo.env.portal.spaceGroup}`);
```
please pay attention to this, we can display the Task application outside a space context which will lead to a null value for `eXo.env.portal.spaceGroup`. You will have to retrieve the group name of the space from space entity assigned to project itself.

@rdenarie FYI

rdenarie replied:
Thanks for this information

rdenarie reviewed (changes requested):
Do not approved this pr until additionnal tests done

IlhemEssaadi committed:
TASK-53005: In Tasks application, updating name project results loss of project. (#752)

ISSUES : In Tasks application, when updating the project name, the updates are not immediately made and the display of the project is lost after refreshing.
FIX : the problem is that the manager and participator variables get the wrong value, so it is fixed by allow manager and participator variables to get the value of profile.originalName which contains the value of GrouId of space.

IlhemEssaadi committed:
Merge branch 'develop' into backport/53005
